packages/query/src/graphql/graphql.module.ts

```diff
@@ -123,7 +120,16 @@ export class GraphqlModule implements OnModuleInit, OnModuleDestroy {
         options.replaceAllPlugins.push(options.appendPlugins.pop());
       }
     }
-    const schema = await this.buildSchema(dbSchema, options, SCHEMA_RETRY_NUMBER);
+
+    if (!argv['disable-hot-schema']) {
+      const pgClient = await this.pgPool.connect();
+      await pgClient.query(`LISTEN "${dbSchema}._metadata.hot_schema"`);
+
+      pgClient.on('notification', () => {
```

Comment:
I think we need to filter here for a specific notification type. Like check if the notification relates to `hot_schema`